tests/typecheck/managers/test_managers.yml

```diff
@@ -543,7 +543,6 @@
         myapp/models:23: error: Could not resolve manager type for "myapp.models.TwoUnresolvable.objects"  [django-manager-missing]
         myapp/models:24: error: Could not resolve manager type for "myapp.models.TwoUnresolvable.second_objects"  [django-manager-missing]
         myapp/models:27: error: Could not resolve manager type for "myapp.models.AbstractUnresolvable.objects"  [django-manager-missing]
-        myapp/models:32: error: Could not resolve manager type for "myapp.models.InvisibleUnresolvable.objects"  [django-manager-missing]
```

Comment:
this makes sense to me at least -- we're no longer redefining an `objects` here because the base class already has an `UnknownManager[Self]` definition

Reply:
cc @flaeppe

Reply:
This is fine IMO. The removal here goes away in my changes in #2252 too